packages/dnb-eufemia/src/components/autocomplete/Autocomplete.js

```diff
@@ -1474,7 +1474,7 @@ class AutocompleteInstance extends React.PureComponent {
 
               if (searchNumbers) {
                 word.split('').forEach((char) => {
-                  if (/[\d\w]/.test(char)) {
+                  if (/[\d\wÆØÅæøå]/.test(char)) {
```

Comment:
```suggestion
                  if (/[\p{L}\p{N}]/u.test(char)) {
```

- [\p{L}\p{N}] matches any Unicode letter (\p{L}) or any Unicode number (\p{N}).
- The u (Unicode) flag enables Unicode-aware matching and allows the use of \p{...} properties.